rest_framework-stubs/response.pyi

```diff
@@ -23,3 +23,6 @@ class Response(SimpleTemplateResponse):
     def render(self) -> Any: ...
     @property
     def status_text(self) -> str: ...
+
+class _MonkeyPatchedResponse(Response):
+    def json(self) -> Any: ...
```

Comment:
I think the other attributes from `_MonkeyPatchedWSGIResponse` are relevant too. Because it seems to re-use the same logic from Django: https://github.com/django/django/blob/9bd174b9a75299dce33e673a559f2b673399b971/django/test/client.py#L818-L824